### DISCUSSION TITLE
Laser suddenly no longer engrave/cut

### DISCUSSION THREAD

**Discussion by edoardoado** [arkypita/LaserGRBL#1131] [Q&A]
Hi, i have a Alfawise C30 Pro 3000mw 445nm 12V by 4 months.

It was working perfectly, but it suddenly stop working properly:
- it doesn't switch on with the command M3S1 (only from S20 or more) and it stay on for few seconds only.
- i can set any speed/power but it will not engrave/cut anything at all (wood/paper doesn't matter)

I didn't upgrade the computer or the software, it just looks like the laser lost power (the lens is clean), software/connection/motors work fine

Thanks

**Comment by StuartB4** [comment]
Sounds like the laser may be overheating and switching off. 
Does the engraver have this control board? 

![conboard2](https://user-images.githubusercontent.com/16980026/94155175-8fc6fc00-fe76-11ea-97ab-0938a2616716.jpg)

**Comment by edoardoado** [comment]
No it use this kind

![IMG_7467](https://user-images.githubusercontent.com/71830373/94165927-08d04e80-fe8b-11ea-9021-9ab1a02c8b3f.jpg)

**Comment by StuartB4** [comment]
Oh ok. Sorry I don't know enough about that board. 
Just curious though, where do the 3 black wires plugged in to the Probe connector go to?

**Comment by edoardoado** [comment]
That's a shock sensor to stop the machine in case it get hit

**Comment by StuartB4** [comment]
Oh ok, thanks.
